docs/sdk/v3/guides/liquidity/02-modifying-position.md

```diff
@@ -0,0 +1,115 @@
+---
+id: liquidity
+title: Adding & Removing Liquidity
+---
+
+## Introduction
+
+This guide will cover how to modify a liquidity position by adding or removing liquidity on the Uniswap V3 protocol.
+It is based on the [modifying a position code example](https://github.com/Uniswap/examples/tree/main/v3-sdk/modifying-position), found in the Uniswap code examples [repository](https://github.com/Uniswap/examples).
+To run this example, check out the examples's [README](https://github.com/Uniswap/examples/blob/main/v3-sdk/modifying-position/README.md) and follow the setup instructions.
+
+:::info
+
+If you need a briefer on the SDK and to learn more about how these guides connect to the examples repository, please visit our [background](./01-background.md) page!
+
+:::
+
+In the Uniswap V3 protocol, liquidity positions are represented using non-fungible tokens. In this guide we will use the `NonfungiblePositionManager` class to help us mint a liquidity position and then modify the provided liquidity for the  **USDC - DAI** pair. The inputs to our guide are the **two tokens** that we are pooling for, the **amount** of each token we are pooling for, the Pool **fee** and the **percentage** by which to **add and remove** from our position.
+
+The guide will **cover**:
+1. Creating or minting our position using the `NonfungiblePositionManager`'s `addCallParameters` to get the data for creating the position minting transaction, and then executing the transaction.
+2. Adding liquidity to our position using the `NonfungiblePositionManager`'s `addCallParameters` to get the data for creating the add liquidity transaction, and then executing the transaction.
+3. Removing our position's liquidity using the `NonfungiblePositionManager`'s `removeCallParameters` to get the data for creating the remove liquidity transaction, and then executing the transaction.
```

Comment:
These titles are a bit unwieldy - can we cut them down and bring them into the body of each section?

1. Creating and minting our position
2. Adding liquidity to our position
3. Removing liquidity from our position

More importantly though, don't we already cover the first step in the first guide? Can we link there instead of spending a whole section repeating?